setupini.sh

```diff
@@ -0,0 +1,103 @@
+                echo "Database is running. No need to start ...."
+        else
+                echo "Database is not running. Starting the database...."
+                start_db=`sudo service postgresql start`
+                if [ $? -eq 0 ]; then
+                        echo "Database started"
+                fi
+        fi
+else
+        sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+        if [ $? -eq 0 ]; then
+                wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+        fi
+        if [ $? -eq 0 ]; then
+                echo "Key added"
+                sudo apt-get update
+        fi
+        if [ $? -eq 0 ]; then
+                echo "Updates of the packages done."
+                sudo apt-get install postgresql
+        fi
+        if [ $? -eq 0 ]; then
+                echo "PostGRE installed. Checking version"
+                version=`psql --version`
+                echo $version
+        fi
+
+
+fi
+
+
+### Python installation check
+
+echo "Checking python installation"
+py_status=$(python3 --version | wc -l)
+py_version=$(python3 --version)
+py_ver=$(echo $py_version | cut -d " " -f2)
+if [ $py_status -ge 1 ]; then
+        echo "Python is installed and version is $py_version"
+        if [$py_ver = *Python 3.9* ]; then
+                echo "Python 3.9+ installed. No further installation needed"
+        else
+                echo "Python 3.9 is needed"
+                sudo apt update
+                if [ $? -eq 0 ]; then
+                        sudo apt install -y software-properties-common
+                        if [ $? -eq 0 ]; then
+                                sudo add-apt-repository -y ppa:deadsnakes/ppa
+                                if [ $? -eq 0 ];then
+                                        sudo apt install -y python3.9
+                                        if [ $? -eq 0 ]; then
+                                                py_latest=$(python3.9 --version)
+                                                echo "Latest Installed Python version is $py_latest"
+                                                py_latest_install="python"$(echo $py_latest | cut -d " " -f2)
+                                                sudo update-alternatives --install /usr/local/bin/python3 python /usr/bin/python3.9 1
+                                                if [ $? -eq 0 ]; then
+                                                        echo "Latest Python version set. It is $(python3 --version)"
+                                                fi
+
+                                        fi
+                                fi
+                        fi
+                fi
+
+        fi
+fi
+
+
+
+### Python packages installation
+
+# Searching requirements.txt in the current path
+find_requirements_file=$(find . -type f -name "requirements.txt")
```

Comment:
I don't think this is necessary. `pip3` will already fail with a descriptive error message if `requirements.txt` isn't found. And since presumably you would have cloned this repository in order to run this script, it shouldn't ever not be found anyway.